README.md

```diff
@@ -133,11 +133,16 @@ Please note for this to work you should create/update `${GRADLE_USER_HOME}/gradl
 ### Determining if any dependencies could be updated ###
     ./gradlew dependencyUpdates
 
+### Running findbugs on the java and scala code ###
+    ./gradlew findbugsMain findbugsTest -x test
+
+The findbugs warnings will be found in `findbugs/main.html` files inside the subproject directories.
+
 ### Running checkstyle on the java code ###
     ./gradlew checkstyleMain checkstyleTest
 
 This will most commonly be useful for automated builds where the full resources of the host running the build and tests
-may not be dedicated to Kafka's build.
+may not be dedicated to Kafka's build.  The checkstyle warnings will be found in `checkstyle/main.html` files inside the subproject directories.
 
```

Comment:
Thanks for the update. I was thinking that it may make sense to have a `Running code quality checks` section which describes `./gradlew check` (I didn't even know this existed until recently) and then we could have subsections for findbugs and checkstyle. Also, for checkstyle, it's worth making it clear that the warnings are printed to the console as well (and will fail the build) whereas findbugs only produces the report.

What do you think?